fbw-common/src/systems/instruments/src/EFB/Settings/Pages/RealismPage.tsx

```diff
@@ -178,6 +181,12 @@ export const RealismPage = () => {
           )}
         </SettingGroup>
       )}
+
+      {airframeInfo.variant === AirframeType.A380_842 && (
```

Comment:
This should instead be via the AircraftContext settings options rather than generic EFB pages (in fbw-common) having knowledge of specific aircraft. You can see a good example of how to do it on the setting immediately above.

Reply:
Didn't know which pattern to use, fixed now